### PR TITLE
Add README.md and images

### PR DESCRIPTION
**Pull request by scottdangelo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,570 @@
-# predict-credit-risk-with-jupyter-on-cloud-pak-for-data
-Use IBM Cloud Pak for Data and Jupyter to predict credit risk
+# Predict Credit Risk With Jupyter on Cloud Pak for data
+
+In this Code Pattern, we will go through the process of exploring a data set and building a predictive model that can be used to determine the likelyhood of a credit loan having 'Risk' or 'No Risk'. For this use case, the machine learning model we are building is a classification model that will return a prediction of 'Risk' (the features of the loan applicant predict that there is a good chance of default on the loan) or No Risk (the applicant's inputs predict that the loan will be paid off). The approach we will take in this lab is to some fairly popular libraries / frameworks to build the model in Python using a Jupyter notebook. Once we have built the model, we will make it available for deployment so that it can be used by others.
```
Comment (rhagarty) [inline README.md]: - likelihood
- last 'No Risk' should have quotes to be consistent
- "in this lab is to some". Missing verb like "use" or "employ"

File: README.md [hunk 2]
```diff
@@ -1,2 +1,570 @@
...
+
+After building the model, we will go through the process of deploying a machine learning model so it can be used by others. Deploying a model allows us to put a model into production, so that data can be passed to it to return a prediction. The deployment will result in an endpoint that makes the model available for wider use in applications and to make business decisions. There are several types of deployments available ([depending on the model framework used](https://www.ibm.com/support/producthub/icpdata/docs/content/SSQNUZ_current/wsj/analyze-data/pm_service_supported_frameworks.html)), of which we will explore:
```
Comment (rhagarty) [inline README.md:5]: first sentence is a repeat of previous sentence

File: README.md [hunk 3]
```diff
@@ -1,2 +1,570 @@
...
+
+#### Load and Prepare Dataset
+
+* Section `1.0 Install required packages` will install some of the libraries we are going to use in the notebook (many libraries come pre-installed on Cloud Pak for Data). Note that we upgrade the installed version of Watson Machine Learning Python Client. Ensure the output of the first code cell is that the python packages were successfully installed.
+
+    * Run the code cells in section 1.1 and 1.2. Ensuring that the cells complete before continuing.
+
+    ![Imported packages](doc/source/images/mljupyter-packages-installed.png)
+
+* Section `2.0 Load and Clean data` will load the data set we will use to build out machine learning model. In order to import the data into the notebook, we are going to use the code generation capability of Watson Studio.
```
Comment (rhagarty) [inline README.md]: "to build our machine"

**Review by rhagarty — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (scottdangelo) [reply README.md]: fixed, thanks.

File: README.md (hunk 2 above)
Comment (scottdangelo) [reply README.md:5]: fixed, thanks.

File: README.md (hunk 3 above)
Comment (scottdangelo) [reply README.md]: fixed, thanks.

**Review by rhagarty — APPROVED** [review]

**Pull request merged**